.travis.yml

```
@@ -39,4 +39,3 @@ deploy:
   on:
     python: 3.7
```

```suggestion
    python: 3.8
```
Let's also deploy using 3.8